src/models/v2/server.ts

```diff
@@ -0,0 +1,43 @@
+import { Mixin } from '../utils';
+import { BaseModel } from '../base';
+import { ServerInterface } from '../server';
+import { DescriptionMixin } from './mixins/description';
+import { BindingsMixin } from './mixins/bindings';
+
+export class Server extends Mixin(BaseModel, DescriptionMixin, BindingsMixin) implements ServerInterface {
+    id(): string {
+        return this.json('id');
+    }
+
+    hasName(): boolean {
+        return !!this.json('name');
+    }
+
+    name(): string | undefined {
+        return this.json('name');
+    }
```

Comment:
@smoya what is the `name` method in the server - https://github.com/asyncapi/parser-api/blob/master/docs/v1.md#server?

Reply:
The `name` is the key in the map of servers.

Reply:
then what is `id` then?

Reply:
> the key in the map of servers

Reply:
ok, so `name` and `hasName` is out @Souvikns

Reply:
So then what is `id`? I was thinking that `id` is the key

Reply:
I think it makes sense to use ID for all objects like Channel, Server, Operation, etc, instead of `path()` for Channels, `name` for Servers, etc
If we add a ID method for all of those models, no matter if we later change the source of them between versions.
I know the parser-api doesn’t reflect that but we learnt a lot during all those months :)